### PR TITLE
Added environment variables for proxy configuration

### PR DESCRIPTION
**Pull request by dusansusic**
Environment variables related to proxy configuration were missing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,15 @@ To configure the installation you can either specify the options on the command
 * instana.agent.endpoint.port
 * instana.zone
 
+Optionally, if your infrastucture uses proxy, you should ensure that you set values for:
```
Comment (cedricziel) [inline README.md]:
```suggestion
Optionally, if your infrastructure uses a proxy, you should ensure that you set values for:
```

File: README.md [hunk 2]
```diff
@@ -54,6 +54,15 @@ To configure the installation you can either specify the options on the command
...
+
+* instana.agent.proxyHost
+* instana.agent.proxyPort
+* instana.agent.proxyProtocol
+* instana.agent.proxyUser
+* instana.agent.proxyPassword
+* instana.agent.proxyUseDNS
+
 Check the values for the endpoint entries in [agent backend configuration](https://docs.instana.io/quick_start/agent_configuration/#backend)
```
Comment (cedricziel) [inline README.md]:
```suggestion
Check the values for the endpoint entries in the [agent backend configuration](https://docs.instana.io/quick_start/agent_configuration/#backend).
```

**Comment by doertedev** [comment]
Eh fabs - we initially tested this once with socks4 and socks5. I just want to reiterate: is this really neccessary anymore especially in K8s environments?

File: templates/daemonset.yaml [hunk 3]
```diff
@@ -26,18 +26,42 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
             - name: INSTANA_AGENT_LEADER_ELECTOR_PORT
-              value: "{{ .Values.instana.leaderElectorPort }}"
+              value: {{ .Values.instana.leaderElectorPort | quote }}
             - name: INSTANA_ZONE
               value: {{ .Values.instana.zone }}
             - name: INSTANA_AGENT_ENDPOINT
```
Comment (cedricziel) [inline templates/daemonset.yaml]: This value is not quoted. Should be quoted?

Comment (dusansusic) [reply templates/daemonset.yaml]: quoted all env variables to make consistent structure

**Pull request merged**